doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -54,7 +54,7 @@ Labs that are hosting multiple *Pods* should utilize a leaf / spine topology whe
 
 Leaf switches must provide interfaces matching the physical server specifications above and northbound (spine connections) of 100 Gbps connections. Spine switches must provide the corresponding 100 Gbps interfaces to each leaf switch.  The minimum requirement is one spine switch.
 
-The minimum networking configuration must provide at least VLANs to partition the various networks required for the reference implementation deployment and separation of each *Pod*, if multiple *Pods* are hosted within the labs.   
+The minimum networking configuration must provide at least VLANs to partition the various networks required for the reference implementation deployment and separation of each *Pod*, if multiple *Pods* are hosted within the labs.
```

Comment:
@markbeierl RI-1 Ch04: As 4.2.2 is describing CNTT networking, it should refer to RA-1 Ch04: [4.2.3.2 High Level Logical Network Layout](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#4232-high-level-logical-network-layout) and the terminology should be consistent. Thanks.